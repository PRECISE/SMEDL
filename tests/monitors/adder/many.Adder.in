measurement,1
measurement,2
measurement,3.5
measurement,4.0
measurement,5.0
