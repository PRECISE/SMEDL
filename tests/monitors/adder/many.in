{
    "fmt_version": [2, 0],
    "channel": "conn",
    "event": "measurement",
    "params": [1],
    "aux": {
        "arbitrary": "json data"
     }
} 
{
    "fmt_version": [2, 0],
    "channel": "conn",
    "event": "measurement",
    "params": [2],
    "aux": [1, 2, 3]
} 
{
    "fmt_version": [2, 0],
    "channel": "conn",
    "event": "measurement",
    "params": [3.5],
    "aux": "foo"
} 
{
    "fmt_version": [2, 0],
    "channel": "conn",
    "event": "measurement",
    "params": [4.0],
    "aux": 36
} 
{
    "fmt_version": [2, 0],
    "channel": "conn",
    "event": "measurement",
    "params": [5.0],
    "aux": {
    }
} 
