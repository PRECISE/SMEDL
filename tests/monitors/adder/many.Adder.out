_Adder_sum,1.0
_Adder_sum,3.0
_Adder_sum,6.5
_Adder_sum,10.5
_Adder_sum,15.5
